gl-zkPass/zkpass-sdk
# update rust doc

how to generate docs for zkpass_client only? => `cargo doc --no-deps --open -r -p zkpass-client`

- Updated cargo doc for zkpass_client

What to test?
1. Make sure the logos are appearing on every pages
2. Make sure there aren't broken text style or any typos (for example, there is a tab on an unusual place)
3. Make sure the theme settings is working properly (dark, light, ayu)
4. Make sure the search feature is working
5. (Optional) Please compare this generated docs with the ones on the development, make sure there is no unmatch styling, wording, etc.

How to test?
1. go to `docs/rust/zkpass_client/index.html`
2. go to every link possible
